packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -65,10 +69,34 @@ const BpkAccordionItem = (props: Props) => {
   // $FlowFixMe[prop-missing] - see above
   delete rest.initiallyExpanded;
 
-  if (expanded) {
+  if (expanded && !onDark) {
     iconClassNames.push(
       getClassName('bpk-accordion__item-expand-icon--flipped'),
     );
+    contentClassNames.push(
+      getClassName('bpk-accordion__content-container--expanded'),
+    );
+  }
+
+  if (expanded && onDark) {
+    iconClassNames.push(
+      getClassName('bpk-accordion__item-expand-icon--flipped'),
+    );
+    iconClassNames.push(getClassName('bpk-accordion__item-expand-icon--day'));
+    contentClassNames.push(
+      getClassName('bpk-accordion__content-container--expanded-day'),
+    );
+    titleTextClassNames.push(getClassName('bpk-accordion__title-text--day'));
+  }
+
+  if (!expanded && onDark) {
+    iconClassNames.push(getClassName('bpk-accordion__item-expand-icon--day'));
+    titleClassNames.push(getClassName('bpk-accordion__title--expanded-day'));
+    titleTextClassNames.push(getClassName('bpk-accordion__title-text--day'));
```

Comment:
If we are applying these when it's `onDark` can you rename it from instead of being called `--day` to be `--on-dark`?

Reply:
Yes, good idea. I was getting carried away with all this day/night stuff.